src/service/video/docking.js

```diff
@@ -1578,6 +1599,39 @@ export class VideoDocking {
     this.placeAt_(video, x, y, scale, step, /* optTransitionDurationMs */ 200);
   }
 
+  /**
+   * @param {number} x
+   * @param {number} y
+   * @return {boolean}
+   * @private
+   */
+  isInsideTargetAreaWhenDocked_(x, y) {
+    const video = this.getDockedVideo_();
```

Comment:
I think it would be simpler if you just walk up the tree from event.target to see if it was in the video player ( and checking isDocked to only handle docked case)

Reply:
Donezo.